core/process.py

```diff
@@ -1777,9 +1777,17 @@ def DisplayJobs(self, style):
         #      24510                       | sleep 5 &
 
         f = mylib.Stdout()
+        current, previous = self.GetCurrentAndPreviousJobs()
         for job_id, job in iteritems(self.jobs):
             # Use the %1 syntax
-            job.DisplayJob(job_id, f, style)
+            extra = ''
```

Comment:
I think it should default to one space so it's aligned?

Reply:
Yes! Nice catch